src/js/profile/mobile/widget/Calendar.js

```diff
@@ -137,15 +157,41 @@
 							setDays = setDays + 1;
 							leftDays = leftDays - 1;
 
-							if (div.innerHTML === defaultToday.toLocaleString()) { // today selected.
+							if (defaultToday > parseInt(div.innerHTML, 10)) {
+								if (fixMonth === todayMonth && Calendar.pastSelection === "true") {
```

Comment:
use `!self.options.pastSelection` instead of `Calendar.pastSelection`

Reply:
done.